tests/data/src/required_by_mixed_capitalization/setup.py

```diff
@@ -0,0 +1,6 @@
+from setuptools import setup
+
+setup(name='simple',
+      version='1.0',
+      install_requires=['required_by_Capitalized==1.0']
```

Comment:
I would call this package `requires_requires_capitalized` (impacting the `name` argument here and the directory name) and have it contain `'requires_Capitalized'` in its `install_requires`, then we can remove `required_by_Capitalized`. There will be a few corresponding changes in `test_show_required_by_packages_requiring_capitalized`.